addons/mail/static/src/models/thread/thread.js

```diff
@@ -552,7 +552,7 @@ function factory(dependencies) {
                     partner_ids: [this.env.messaging.currentPartner.id],
                     context: {}, // FIXME empty context to be overridden in session.js with 'allowed_company_ids' task-2243187
                 },
-            }));
+            }, { shadow: true }));
```

Comment:
I would revert this one, this is a direct user action for which we don't handle "loading" ourselves.